Assets/HoloToolkit/Utilities/Scripts/Editor/SceneSettingsWindow.cs

```diff
@@ -95,6 +95,7 @@ protected override void ApplySettings()
                 }
 
                 PrefabUtility.InstantiatePrefab(AssetDatabase.LoadAssetAtPath<GameObject>(AssetDatabase.GUIDToAssetPath(InputSystemPrefabGUID)));
+                FocusManager.Instance.UpdateCanvasEventSystems();
```

Comment:
Do we have added documentation anywhere saying that this setting now changes all my canvases?

Reply:
Nope I need to update ALL the docs 😅

Reply:
Yeah, they need an update soon for sure.

There's nothing in the scene settings window that tells me that this option will change all my canvases. I'd say this should be a new option in the window, because I don't necessarily want my canvases all updated just to put the InputManager prefab into the scene.

Reply:
Ehh, Idk. You def want this change, or else none of your canvases will work.

Reply:
There are cases where you want read only canvases in any UI. Just so long as we have a plan for "Interactable" canvases and non-interactable canvases, we should be good.

Reply:
If you don't want an interactive canvas, simply remove the event camera from it.

Reply:
@DDReaper my latest commits have made this easier to do by asking users if they want to update their world camera to the RaycastCamera.